mubench.docs/mubench.py

```diff
@@ -0,0 +1,26 @@
+from argparse import HelpFormatter, ArgumentParser
+from operator import attrgetter
+
+import sys
+
+
+__TASKS = ['pipeline', 'browse', 'reviewsite', 'update']
+
+
+class SortingHelpFormatter(HelpFormatter):
```

Comment:
Does this formatter actually sort mandatory arguments? Sorting by `option_strings` appears to me like it shouldn't because there are no option strings for mandatory arguments... (this is not strictly related to this task, but might be a general problem with our CLI help)

Reply:
Seems like it doesn't:
```
usage: ./mubench [-h]
                 {check,info,checkout,compile,run,publish,stats,browse} ...

Run MUBench, the benchmark for API-misuse detectors.

positional arguments:
  {check,info,checkout,compile,run,publish,stats,browse}
                        MUBench provides several tasks. Run `./mubench <task>
                        -h` for details.
    check               Check MUBench runtime requirements or dataset
                        consistency.
    info                Show info about projects, project versions, and
                        misuses in MUBench.
    checkout            Checkout project versions.
    compile             Compile project versions. Runs `checkout`, if
                        necessary.
    run                 Run an experiment. Runs `checkout` and `compile`, if
                        necessary.
    publish             Publish data to a review site. Runs `checkout`,
                        `compile`, and `run` , if necessary.
    stats               Print statistics on the MUBench dataset.
    browse              Open a Linux shell in a container mounting the MUBench
                        Docker Volumes.

optional arguments:
  -h, --help            show this help message and exit

For details, check out https://github.com/stg-tud/MUBench.
```

Reply:
I created a respective issue #360.